README.md

```diff
@@ -100,6 +100,21 @@ _Note: SwiftGen needs Xcode 8.3 to build, so installing via Homebrew requires yo
 
 ---
 </details>
+<details>
+<summary>Via <strong>Mint</strong></summary>
+
+To install SwiftGen via [Mint](https://github.com/yonaskolb/Mint), simply use:
+
+```sh
+$ brew install libxml2
+$ mint install SwiftGen/SwiftGen --prevent-global
+```
+
+Note: Will work for Releases > 5.3.0 only.
```

Comment:
I'd change this line to "SwiftGen 6.0 or higher" to avoid people reading too quickly over it.